docker/tidb-bench/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.12.1-alpine3.9 AS goycsb
+
+RUN apk add git build-base \
+  && git clone https://github.com/pingcap/go-ycsb $GOPATH/src/github.com/pingcap/go-ycsb \
+  && cd $GOPATH/src/github.com/pingcap/go-ycsb \
+  && make
+
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+COPY --from=pingcap/tidb-sysbench /usr/local/bin/sysbench /usr/local/bin/sysbench
+COPY --from=pingcap/tidb-sysbench /usr/local/share/sysbench /usr/local/share/sysbench
+
+COPY --from=dmonakhov/alpine-fio /usr/local/bin/fio /usr/local/bin/fio
```

Comment:
Can we just copy this binary to make it work? Is the fio a static binary or dynamically linked binary? From its [Dockerfile](https://hub.docker.com/r/dmonakhov/alpine-fio/dockerfile) I don't see any special instruction for static build.

Reply:
I haven't tested this yet.